frontend/src/css/cfpstyles.css

```diff
@@ -0,0 +1,33 @@
+.cfp-background {
+    background: #0CAEE7;
+}
+.cfp-deco-title {
+    position: relative;
+    padding-bottom: 20px; 
+}
+.cfp-deco-title::before {
```

Comment:
I kind of don't understand this style title. Why deco and what ::before stands for? It's not a change request. I just wonder about style naming convention.

Reply:
"Deco" as "Decorate", ::before flag means that we're generating a pseudo element(in this case horizontal line) connected to HTML element (in this case title "O NAS").